technologies.json

```diff
@@ -458,6 +459,7 @@
       "IL-rewrite",
       "acceptance",
       "assertions",
+      "automation",
```

Comment:
I wonder why this line was added. Could you re-generate technologies.json?

Reply:
Done! 
thx